ginga/util/toolbox.py

```diff
@@ -89,4 +101,59 @@ def _configure_cb(self, view, width, height):
         mode, modetype = bm.current_mode()
         self.mode_change_cb(bm, mode, modetype)
 
+
+def generate_cfg_example(config_name, **kwargs):
+    """Generate config file documentation for a given config name.
+
+    If found, it will be a Python code block of the contents.
+    If not found, it will have a generic message that the config
+    is not available.
+
+    Parameters
+    ----------
+    config_name : str
+        Config name that is attached to the configuration file.
+        This is the same as input for ``prefs.createCategory()``.
+        For example, ``'general'``, ``'channel_Image'``, or
+        ``'plugin_Zoom'``.
+
+    kwargs : dict
+        Optional keywords for :func:`~astropy.utils.data.get_pkg_data_contents`.
+
+    Returns
+    -------
+    docstr : str
+        Docstring to be inserted into documentation.
+
+    """
+    cfgpath = 'examples/configs'  # Where it is in pkg data
+    cfgname = config_name + '.cfg'
+
+    try:
+        cfgdata = get_pkg_data_contents(
+            os.path.join(cfgpath, cfgname), **kwargs)
+    except Exception as e:
+        warnings.warn(str(e), AstropyUserWarning)
+        return ''
+
+    userpath = '~/.ginga'  # Where user should save it
```

Comment:
This may cause problems on some plaftorms (e.g. Windows).  Also, user can override the default home directory location by defining `GINGA_HOME`.  So the proper way to do this is:
```python
from ginga.util.paths import ginga_home
```
Then `ginga_home` is the desired directory prefix.